backend/requirements.txt

```diff
@@ -1,36 +1,54 @@
-alembic==1.8.1
-astroid==2.12.11
+#
+# This file is autogenerated by pip-compile with python 3.8
+# To update, run:
+#
+#    pip-compile
+#
```

Comment:
Will the setup process for backend be different now?

Reply:
Nope, you'll still install the requirements the same way. I installed a package locally that inspects the code and autogenerates the requirements based on what's actually used in the code. Pretty neat!

https://github.com/jazzband/pip-tools

Reply:
> 
This was the exact stackoverflow answer I used lol
https://stackoverflow.com/a/69081814